### PR TITLE
create optimizer config and scheduler config separately

### PR DESCRIPTION
**Pull request by yaoyaowd**
Create individual configs so that we don't need to import cfg from individual module.
It provides better code isolation, config isolation, and safty.
In the future we can easily switch configuration system from the main python file.

I will do it for each module in separated PRs.

### CODE REVIEW

File: graphgym/main.py [hunk 1]
```diff
@@ -12,15 +12,31 @@
                                              update_out_dir, get_parent_dir)
 from torch_geometric.graphgym.loader import create_loader
 from torch_geometric.graphgym.logger import setup_printing, create_logger
-from torch_geometric.graphgym.optimizer import create_optimizer, \
-    create_scheduler
+from torch_geometric.graphgym.optimizer import *
```
Comment (rusty1s) [inline graphgym/main.py:15]: Not a big fan of star imports. I suggest to revert this change.

**Review by rusty1s — APPROVED** [review]
LGTM. Thanks @yaoyaowd

File: graphgym/main.py (hunk 1 above)
Comment (yaoyaowd) [reply graphgym/main.py]: updated.
